effekt/jvm/src/test/scala/effekt/RecursiveDescentTests.scala

```diff
@@ -542,7 +542,7 @@ class RecursiveDescentTests extends munit.FunSuite {
       parseDefinition(
         """def foo = f
           |""".stripMargin),
-      DefDef(IdDef("foo"), None, Var(IdRef(Nil, "f"))))
+      DefDef(IdDef("foo"), None, Var(IdRef(Nil, "f")), None))
 
```

Comment:
Maybe adding a similar test here, with an actual doc comment, would be an option?